test/shopify_app/carrier_service_verification_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'test_helper'
+require 'action_controller'
+require 'action_controller/base'
+
+class CarrierServiceVerificationController < ActionController::Base
+  include ShopifyApp::CarrierServiceVerification
+  def carts_update
```

Comment:
carts_update is an actual webhook name though which is why it was named so. Could you name this endpoint `callback` since that seems closest to how a carrier service would be used.